pages/whats-new/tinacms.tsx

```diff
@@ -48,14 +49,22 @@ const Tinacms = ({ items }) => {
           ))}
         </div>
         <div className="font-tuner text-lg text-center text-blue-700">
-          <Link href="https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md">
-            {' '}
-            See More{' '}
+          <Link
+            href="https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md"
+            className="flex items-center justify-center"
+          >
+            See more on GitHub <FaGithub className="ml-2" />
+          </Link>
+          <Link
+            href="https://us20.campaign-archive.com/home/?u=1fea337bee20e7270d025ea8a&id=c1062536a1"
+            className="flex items-center justify-center"
+          >
+            See Newsletters <FaNewspaper className='ml-2' />
```

Comment:
Not a big one, but in the future I would prefer avoiding hardcoded URLs like this - They should be editable in TinaCMS

Reply:
🫡